src/Surfnet/StepupMiddleware/CommandHandlingBundle/Configuration/CommandHandler/InstitutionConfigurationCommandHandler.php

```diff
@@ -85,6 +87,14 @@ public function handleReconfigureInstitutionConfigurationOptionsCommand(
             new ShowRaaContactInformationOption($command->showRaaContactInformationOption)
         );
 
+        $allowedSecondFactors = array_map(function ($allowedSecondFactor) {
```

Comment:
Nit: I'd pull up the decoding of a list to the top of the method (before loading the aggregate) to get a bit more structure in the code, now we're mixing a mapping of values with the invocation of methods. Why not for the single-value object (the options above)? Because they're more readable (can be inlined in the method call) than a 3 line mapping between two method calls :wink: